sql/report_queries/volume_count_brief/README.md

```diff
@@ -1,2 +1,5 @@
-# ACRL Volume Count Report
+This report is superseded by the [ACRL Item Count (Volume count)](https://github.com/NancyBolduc/folio-analytics/tree/master/sql/report_queries/item_count) query. It was created in the early stage of FOLIO report writing and is no longer officially used for the ACRL Volume Count report. 
```

Comment:
Same here. I'd suggest to change it to  `../item_count`

Reply:
I also changed that one.